qir/qsharp/NegativeBaseProfiles/IfStatement/QSharpVersion/Example.qs

```diff
@@ -0,0 +1,18 @@
+namespace IfCase {
```

Comment:
Here, the Q# uses the accurate name `IfCase` but the test that seems to correlate uses the name `IfWithPhi` and QIR snippet that has `LoopCase` in the function name. This could cause someone trouble later if they try to use this file to regenerate the QIR for that test case.

Reply:
Name updated.